misc/ballerina-bindgen/src/main/java/org/ballerinalang/bindgen/command/BindingsGenerator.java

```diff
@@ -88,9 +88,9 @@
     private static Set<JError> exceptionList = new HashSet<>();
     private static Map<String, String> failedClassGens = new HashMap<>();
 
-    public BindingsGenerator(PrintStream stream) {
-        this.outStream = stream;
-        this.errStream = stream;
+    public BindingsGenerator(PrintStream out, PrintStream err) {
```

Comment:
Can't we make this `package-private` since it is only used in one place inside package.

Reply:
Done